src/model.js

```diff
@@ -240,13 +240,7 @@ export class Model {
     let head = {};
 
     // Add id if no primary key was manually added to definition
-    // Can't use this.primaryKeys here, since this function is called before PKs are identified
-    if (!_.some(this.rawAttributes, 'primaryKey')) {
-      if ('id' in this.rawAttributes) {
-        // Something is fishy here!
-        throw new Error(`A column called 'id' was added to the attributes of '${this.tableName}' but not marked with 'primaryKey: true'`);
-      }
-
+    if (!_.some(this.rawAttributes, 'primaryKey') && !this.options.noPrimaryKey) {
```

Comment:
```suggestion
    if (!this.options.noPrimaryKey && !_.some(this.rawAttributes, 'primaryKey')) {
```

better performances even though it's unnoticeable because this code runs rarely